windows-apps-src/security/companion-device-unlock.md

```diff
@@ -68,6 +71,8 @@ Any number of these user signals can be combined into one. User presence and int
 
 ### Registration and future communication between a PC and Windows Hello companion devices
 
+> In the next major release of Windows following 1809, there is a behavioral change to companion device registration. For users that currently have provisioned and are using a companion device and upgrade to this release, there will be no changes. For users that are not using the companion device framework and upgrade, policy must be enabled before setting up a companion device with Windows Hello.  

```

Comment:
Was this intended to be a warning like the previous change? If so, it's missing `[!WARNING]

Reply:
It's meant to be informational. I think maring this as a Warning would be acceptable.